package.json

```diff
@@ -35,7 +35,7 @@
   },
   "readmeFilename": "README.md",
   "scripts": {
-    "test": "npm run lint && node tests/all && node test/all",
+    "test": "npm run lint && metatests test/ && metatests tests/",
```

Comment:
Fix to: `"test": "npm run lint && metatests test/ && node tests/all"` because `/test/` is migrated to `0.2` but `/tests/` isn't migrated and this PR results in running all files including performance tests in random order.